docs/how-to-install-upgrade-downgrade-kubectl-helm.md

```diff
@@ -0,0 +1,167 @@
+This page shows you how to install, upgrade and downgrade kubectl and helm on macOS, Linux (Ubuntu) and Windows via the packages:
+
+- Homebrew on macOS
+- Snap on Linux (Ubuntu)
+- Chocolatey on Windows
+
+# Requirements:
+
+- homebrew available on macOS
+- chocolatey available on Windows
+- Ubuntu is preferred Linux distro

```

Comment:
Canonical introduced a new packaging system called Snap packaging in Ubuntu 16.04 LTS. It claims to be more stable and secure.
need at least ubuntu 16.04 for snap available